docs/user/customization.md

```diff
@@ -226,11 +226,12 @@ The `manifest-templates` target will output the unrendered manifest templates in
 **IMPORTANT**:
 
 - These customizations require using the `manifests` target that does not provide compatibility guarantees.
-- This can affect upgradability of your cluster as the `machine-config-operator` can mark clusters tainted when user defined [MachineConfig][machine-config] objects are present in the cluster.
 
 In most cases, user applications should be run on the cluster via Kubernetes workload objects (e.g. DaemonSet, Deployment, etc). For example, DaemonSets are the most stable way to run a logging agent on all hosts. However, there may be some cases where these workloads need to be executed prior to the node joining the Kubernetes cluster. For example, a compliance mandate like "the user must run auditing tools as soon as the operating system comes up" might require a custom systemd unit for an auditing container in the Ignition config for some or all nodes.
 
-The configuration of machines in OpenShift is controlled using `MachineConfig` objects and what configuration is applied to a machine in the OpenShift cluster is based on the [MachineConfigPool][machine-config-pool] objects. To allow customization of machine configuration which is not possible as Day 2 operation, the installer allows users to bring their own custom `MachineConfig` objects.
+Further, some aspects of RHEL CoreOS machines (usually kernel arguments such as `nosmt` for disabling hyperthreading) may need to be configured before user workloads land on a system.
```

Comment:
Maybe move the sentence from below: "For these "day 1" cases, `MachineConfig` objects can be provided as additional manifests.` to after this sentence?

Reply:
If you do use nosmt, maybe link to the MCO's docs : https://github.com/openshift/machine-config-operator/blob/5795bb4f2f7b3adf3cf3dfb4100774244a1b8f99/docs/MachineConfiguration.md#nosmt